model.js

```diff
@@ -60,28 +60,50 @@ function translate (input) {
 
     return {
       type: 'FeatureCollection',
-      features: [ formatFeature(input) ],
+      features: [formatFeature(input)],
       metadata: {
         geometryType: geometry
       }
     }
   } else {
+    const features = [];
+    input.features.map((feature) => {
+      features.push(formatFeature(feature))
+    })
+
 
     // Or it's a feature collection
-    return input;
+    return {
+      type: 'FeatureCollection',
+      features: features
+    }
   }
 }
 
-function formatFeature (geometry) {
-
-  const feature = {
-    type: 'Feature',
-    properties: {
-      "type": geometry.type
-    },
-    geometry: geometry
+function formatFeature (input) {
+  let feature = {};
+  if(input.type == "Feature") {
+    feature = input;
+    if(feature.geometry === undefined || feature.geometry === null) {
+      feature.geometry = {
+        type: "Point",
+        coordinates: [ 0, 0]
```

Comment:
What are your thoughts on setting this as `coordinates: [ null, null]`.  We've made some adjustments to winnow to handle null coordinates.  While not valid GeoJSON, this still loads in AGOL and ArcGIS Pro.  No point is rendered for the feature with null coordinates, but it does appear in the attribute table.

Reply:
still kind of a hack, ideally we'd be able to handle `undefined` here

Reply:
I took some sample data and removed geometry so as to simulate feature.geometry: undefined. Passed through FeatureServer and winnow without any observable problems. However, the result in Pro was undesirable - even if only one feature had geometry undefined, the whole dataset was loaded as a table rather than a layer. And attribute table wouldn't load. I found the best all around behavior in Pro with coordinates: [ null, null].

Reply:
ok...oh Pro, why you so fickle?

Reply:
do you think we should make the data conform inside Winnow or throw a warning somewhere?

Reply:
We could both, that way providers wouldn't have to do this kind of transformation, but then there would be a paper trail should questions arise.

Reply:
works for me